docs/usermanual.md

```diff
@@ -521,6 +524,11 @@ Content-type: application/json
 }
 ```
 
+#### Time processing
+
+HTTP bindig is returning in a HTTP header named `X-Processing-Time` processing time expended by current HTTP measure
```

Comment:
In which unit is provide (seconds, etc.)? Are decimal allowed?

Including an example would be great.